connector_importer/models/record.py

```diff
@@ -71,14 +72,14 @@ def debug_mode(self):
         self.ensure_one()
         return self.backend_id.debug_mode or os.environ.get("IMPORTER_DEBUG_MODE")
 
-    def import_record(self, importer_config):
+    def import_record(self, importer_config_dict):
         """This job will import a record.
 
-        # TODO rewrite
-        :param component_name: name of the importer component to use
-        :param model_name: name of the model to import
-        :param is_last_importer: flag for last importer of the recordset
+        :param importer_config: configuration of the importer
         """
+        # set the importer_config as DotDict
+        importer_config = DotDict(importer_config_dict)
```

Comment:
when the dict arrives here should be already a DotDict

Reply:
that's not the case and I was not able to run the importer because type was `dict`. maybe the serialization/unserialization has something to do here

Reply:
yeah, let me rephrase: please make sure we get always a DotDict before we get here :wink:

Reply:
this should come only from https://github.com/OCA/connector-interfaces/blob/14.0/connector_importer/models/import_type.py#L120 and as such, we should always have DotDict here.
If not, then we probably need to call it at the end of `_make_importer_info` instead.